backend/src/test/java/botobo/core/auth/AuthAcceptanceTest.java

```diff
@@ -0,0 +1,44 @@
+package botobo.core.auth;
+
+import botobo.core.AcceptanceTest;
+import botobo.core.auth.dto.GithubUserInfoResponse;
+import botobo.core.auth.dto.LoginRequest;
+import botobo.core.auth.dto.TokenResponse;
+import botobo.core.auth.infrastructure.GithubOauthManager;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+public class AuthAcceptanceTest extends AcceptanceTest {
+
+    @MockBean
+    private GithubOauthManager githubOauthManager;
+
+    public TokenResponse 로그인되어_있음() {
```

Comment:
```suggestion
    protected TokenResponse 로그인되어_있음() {
```
상속구조를 사용한 김에 확실하게 하면 코드가 멋질 것 같아요 😁